monkey/monkey_island/readme.md

```diff
@@ -0,0 +1,98 @@
+>To easily setup development environment for Monkey Island and the Monkey look into deployment scripts folder.
+
+>If you want to setup dev. env. for Island manually, refer to the instructions below.
+
+## How to set up the Monkey Island server:
+
+### On Windows:
+0. Exclude the folder you are planning to install the Monkey in from your AV software, as it might block or delete files from the installation.
+1. Create folder "bin" under monkey_island
+2. Place portable version of Python 3.7.4
```

Comment:
Why are we doing a venv here and not with the monkey itself?

Reply:
No clue why it was written that way, but since deployment scripts don't use venv it would make sense to remove it here. What do you think?